Doc/whatsnew/3.13.rst

```diff
@@ -815,7 +815,7 @@ Build Changes
 =============
 
 * Autoconf 2.71 and aclocal 1.16.4 is now required to regenerate
-  :file:`!configure`.
+  the ``configure`` script.
```

Comment:
No reason to remove the markup here, AFAICT:
```suggestion
  the :file:`!configure` script.
```

Reply:
Check how it is rendered: with the exclamation mark.

Reply:
Ahh! No cross-reference is generated, so we should just remove the ``!``. Updated the suggestion. Ref: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-file

Reply:
I modified my PR to use the ``:file:`configure` `` syntax